pkgs/tools/networking/popura/default.nix

```
@@ -0,0 +1,25 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "popura";
+  version = "0.4.6";
+
+  src = fetchFromGitHub {
+    owner = "${pname}-network";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-iCu6/vD4vgn7aGdwK+OB8ib/QwUwoFuxDUs7vqbTZQc=";
+  };
+
+  vendorHash = "sha256-9lQC35yt1S2uch3qgwNfa/1FHy+Qi1D5Jo7DWNMgU9w=";
+
+  meta = with lib; {
+    description = "Popura ポプラ: alternative Yggdrasil network client";
+    homepage = "https://popura-network.github.io";
```

the original link doesn't work for me
```suggestion
    homepage = "https://github.com/popura-network/popura";
```

---

hmm, it does look broken; sad